src/modules/battery_status/analog_battery.cpp

```diff
@@ -40,8 +40,8 @@ AnalogBattery::AnalogBattery(int index, ModuleParams *parent) :
 }
 
 void
-AnalogBattery::updateBatteryStatusADC(hrt_abstime timestamp, float voltage_raw, float current_raw,
-				      bool selected_source, int priority, float throttle_normalized)
+AnalogBattery::updateBatteryStatusADC(hrt_abstime timestamp, int32_t voltage_raw, int32_t current_raw,
```

Comment:
Did you really intend to change to int32_t?

Reply:
Huh, I did not fix that merge conflict correctly, thanks!